srcpkgs/umurmur/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'umurmur'
 pkgname=umurmur
 version=0.2.20
-revision=1
+revision=2
 build_style=gnu-configure
-configure_args="--enable-shmapi --with-ssl=openssl"
```

Comment:
i'd vote do explicitly disable and link related bug reports at least in the commit